packages/tangle/tangle_test.go

```diff
@@ -232,17 +239,20 @@ func TestTangle_MissingMessages(t *testing.T) {
 	// setup the message factory
 	tangle.MessageFactory = NewMessageFactory(
 		tangle,
-		TipSelectorFunc(func(p payload.Payload, countStrongParents, countWeakParents int) (strongParents, weakParents MessageIDs, err error) {
-			r := tips.RandomUniqueEntries(countStrongParents)
+		TipSelectorFunc(func(p payload.Payload, countParents int) (parentsMessageIDs MessageIDs, err error) {
+			r := tips.RandomUniqueEntries(countParents)
 			if len(r) == 0 {
-				return []MessageID{EmptyMessageID}, []MessageID{}, nil
+				return []MessageID{EmptyMessageID}, nil
 			}
 			parents := make([]MessageID, len(r))
 			for i := range r {
 				parents[i] = r[i].(MessageID)
 			}
-			return parents, []MessageID{}, nil
+			return parents, nil
 		}),
+		func(parents MessageIDs, tangle *Tangle) (MessageIDs, error) {
+			return []MessageID{}, nil
+		},
```

Comment:
This is repeated a bunch of times. Maybe it makes sense to make it reusable in `testutil`?